std/array.d

```diff
@@ -1360,6 +1363,84 @@ pure nothrow bool sameTail(T)(in T[] lhs, in T[] rhs)
     }}
 }
 
+/**
+Checks whether a `slice` is part of a `whole` array.
```

Comment:
I would have used `part` instead of `slice`.

Reply:
Ok renamed.